robosats/middleware.py

```diff
@@ -8,7 +8,6 @@
 from django.utils.deprecation import MiddlewareMixin
 from django.http import JsonResponse
 from rest_framework.authtoken.models import Token
-from rest_framework.exceptions import AuthenticationFailed
```

Comment:
Removed this line to fix the lint check [error](https://github.com/RoboSats/robosats/actions/runs/9549559934/job/26319402405?pr=1326)

```console
robosats/middleware.py:11:39: F401 [*] `rest_framework.exceptions.AuthenticationFailed` imported but unused
```

because the import is not used, but it failed with the same message in the run of the commit that removed it 🤔

Reply:
It's a linting error that made it to `main` on a merge conflict resolution. Should now be fine if this branch is updated.